test/test_utils.py

```diff
@@ -97,7 +97,7 @@ def __init__(self):
 
             def forward(self, input_var):
                 self.counter += 1
-                return input_var
+                return input_var * input_var
```

Comment:
The second forward pass is only done if actually needed.
(Performance gain, yay!)
We return input_var * input_var to force saving a tensor on the graph.